config/default.schema.json

```diff
@@ -342,6 +342,28 @@
 				}
 			}
 		},
+		"OIDCMOCK": {
```

Comment:
This var used for seeding only. Is it necessary to define it like this? For my understanding this file exclusively defines vars that the server needs to run.

Reply:
I had a brief discussion about this with @CeEv. Adding variables here is preferred for the sake of documentation. Usage of the variables is indeed confusing, tho. We have multiple ways of adding variables (via schema*json, globals.js, environmental variables; also accessing those via nest service or the sc-commons lib). Moreover, some of which are broken (default values in the schema). 

In our case, this is just making local development adjustments clearer (compared to fiddle around environmental variables).

Reply:
OK, I agree. 
In that case I suggest to add to the description that this is only used during seeding. The current description kind of implies it, but it does not say it explicitly.

Reply:
agreed